src/blog/2024/01/unified-namespace-what-broker.md

```diff
@@ -0,0 +1,50 @@
+---
+title:  Selecting a broker for your Unified Namespace
+subtitle: The broker is the data backbone for the unified namespace, which one is right for you?
+description: The broker is the data backbone for the unified namespace, which one is right for you?
+date: 2024-01-16
```

Comment:
@gdziuba We can swap the dates of this post and https://github.com/FlowFuse/website/pull/1520. What do you think? Than this one goes tomorrow, giving us more time to review.

Reply:
@ZJvandeWeg That works for me.  Do you need me to adjust anything?

Reply:
As it's already the 17th, I'll push this one out to the 19th

Reply:
```suggestion
date: 2024-01-19
```